### ISSUE TITLE
Icon request: Bot

### ISSUE THREAD

**Issue by mittalyashu** [feathericons/feather#688]
## Icon Request

* Icon name: Bot
* Use case: Showing some kind of computing stuff done by bot

I have created this bot image _(considering feather icon guidelines)_

![bot](https://user-images.githubusercontent.com/29014463/61593618-72a5f600-abff-11e9-89fb-140db70b23e3.png)

**Comment by locness3** [comment]
Go ahead, follow #171 to submit it!

**Comment by mittalyashu** [comment]
I did follow the Feather icon design guidelines, but I need some feedback on improving this icon.

**Comment by locness3** [comment]
I think it's okay. Any feedback @johnletey @ahtohbi4

**Comment by ahtohbi4** [comment]
@mittalyashu could you provide two kinds of screenshots: the bigger one with Figmas pixel grid and the second one in a set of other icons? It helps to see all the nuances at once.

[Example](https://github.com/feathericons/feather/issues/661#issuecomment-508099766).

**Comment by johnletey** [comment]
@mittalyashu I personally like the second one better!

**Comment by mittalyashu** [comment]
Here's the bigger view of Bot icon

![Figma_xwS9NmYW2L](https://user-images.githubusercontent.com/29014463/61684998-ebf53400-ad38-11e9-8bfc-3a8290b2de08.png)

_(personally don't like this icon much 😅)_

**Comment by johnletey** [comment]
@mittalyashu Give the robot a smile!

**Comment by mittalyashu** [comment]
Sure, how about now?

![image](https://user-images.githubusercontent.com/29014463/61687981-645ff300-ad41-11e9-881c-ae91875b7ef8.png)

**Comment by locness3** [comment]
Great! Can you do what @ahtohbi4 said ?

**Comment by mittalyashu** [comment]
@locness3 I don't know how to create the second screenshot.

**Comment by johnletey** [comment]
@mittalyashu Give me the SVG code (for both versions) ... and I'll do it for you 😄!

**Comment by mittalyashu** [comment]
Here's the link to [Figma file](https://www.figma.com/file/eBL2HsKSuUOrhXB3vbTCKb/Feather-Bot-Icon?node-id=0%3A1) @johnletey

**Comment by johnletey** [comment]
@locness3 @ahtohbi4 @mittalyashu 

![Shot 2019-07-23 at 08 19 32](https://user-images.githubusercontent.com/30328854/61691072-a9c0f800-ad22-11e9-99c1-47970f4c71b5.jpg)

I'd suggest decreasing the size of the eyes:

![Shot 2019-07-23 at 08 22 39](https://user-images.githubusercontent.com/30328854/61691273-189e5100-ad23-11e9-947f-326aaaf46479.jpg)

```svg
<svg
  width="24"
  height="24"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <rect x="3" y="7" width="18" height="14" rx="2" />
  <line x1="7" y1="11" x2="7" y2="11" />
  <line x1="17" y1="11" x2="17" y2="11" />
  <line x1="6" y1="7" x2="6" y2="3" />
  <line x1="18" y1="7" x2="18" y2="3" />
  <path d="M8 15s1.5 2 4 2 4-2 4-2" />
</svg>
```

**Comment by ahtohbi4** [comment]
> @locness3 I don't know how to create the second screenshot.

I wrote about it [here](https://github.com/feathericons/feather/issues/661#issuecomment-508108770).

**Comment by mittalyashu** [comment]
> I wrote about it [here](https://github.com/feathericons/feather/issues/661#issuecomment-508108770).

Thanks.

**Comment by mittalyashu** [comment]
> (personally don't like this icon much 😅)

We need to make improvements to that bot icon.

**Comment by johnletey** [comment]
@mittalyashu Just a small update ... I moved the eyes to be aligned with the edges of the mouth:

![Shot 2019-07-23 at 10 59 37](https://user-images.githubusercontent.com/30328854/61703210-0a5b2f80-ad39-11e9-956e-482d549fd730.jpg)

```svg
<svg
  width="24"
  height="24"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <rect x="3" y="7" width="18" height="14" rx="2" />
  <line x1="8" y1="11" x2="8" y2="11" />
  <line x1="16" y1="11" x2="16" y2="11" />
  <line x1="6" y1="7" x2="6" y2="3" />
  <line x1="18" y1="7" x2="18" y2="3" />
  <path d="M8 15s1.5 2 4 2 4-2 4-2" />
</svg>
```

**Comment by mittalyashu** [comment]
His eyes are missing 👀...

![image](https://user-images.githubusercontent.com/29014463/61703721-d0515400-ad5f-11e9-9742-cf135f117b6e.png)

**Comment by mittalyashu** [comment]
Does this looks like a bot?

![image](https://user-images.githubusercontent.com/29014463/61704012-6be2c480-ad60-11e9-95fb-04f82a3eda30.png)

@locness3 @ahtohbi4 @johnletey

**Comment by johnletey** [comment]
> His eyes are missing 👀...

Yes ... a Figma bug ... it doesn't display those points ... but the SVG itself is fine!

> Does this look like a bot? Maybe with bigger eyes?

I like that a lot more than the previous one!

**Comment by mittalyashu** [comment]
Even though I also liked the second bot.

But still, we should keep both of them, who knows we might need in the future.

@johnletey Can you also create those screenshots for the second bot?

**Comment by johnletey** [comment]
@mittalyashu Here you go!

![Shot 2019-07-24 at 10 41 01](https://user-images.githubusercontent.com/30328854/61783430-b0716d00-adff-11e9-8989-dd59daf74f98.jpg)

`bot-2` SVG code:

```svg
<svg
  width="24"
  height="24"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <rect x="3" y="11" width="18" height="10" rx="2" />
  <circle cx="12" cy="5" r="2" />
  <path d="M12 7v4" />
  <line x1="8" y1="16" x2="8" y2="16" />
  <line x1="16" y1="16" x2="16" y2="16" />
</svg>
```

**Comment by mittalyashu** [comment]
@johnletey You were trying to increase second bot eyes, how did it go?

**Comment by johnletey** [comment]
> @johnletey You were trying to increase second bot eyes, how did it go?

I was just playing around with the eyes on the first bot (with the smile) ... didn't go so well though ... I'm still not a fan of the way the first bot looks right now

**Comment by ahtohbi4** [comment]
I would stop on the `bot` version from [the comment](https://github.com/feathericons/feather/issues/688#issuecomment-514560575). I like it! 👍

**Comment by mittalyashu** [comment]
> I was just playing around with the eyes on the first bot (with the smile) ... didn't go so well though ... I'm still not a fan of the way the first bot looks right now

Can you try to increase the eye size of the second bot?

> I would stop on the bot version from the comment. I like it! 👍

I kinda liked both the bots 😅

**Comment by mhosman** [comment]
+1 for the bot icon

**Comment by shiv07tiwari** [comment]
Is someone implementing this?

**Comment by moeenio** [comment]
FYI, this is available in Lucide, a fork of Feather icons : https://lucide.dev/icons/bot
